stores/migrations.go

```diff
@@ -1032,16 +1032,6 @@ func performMigration00019_accountsShutdown(txn *gorm.DB, logger *zap.SugaredLog
 
 func performMigration00020_missingIndices(txn *gorm.DB, logger *zap.SugaredLogger) error {
 	logger.Info("performing migration 00020_missingIndices")
-	var err error
```

Comment:
I would maybe drop a comment here to explain what happened. It looks weird without any context.